backend/src/nodes/impl/upscale/convenient_upscale.py

```diff
@@ -68,7 +68,7 @@ def convenient_upscale(
             upscale(as_target_channels(white, model_in_nc, True)), 3, True
         )
 
-        alpha = 1 - np.mean(white_up - black_up, axis=2)
+        alpha = 1 - np.mean(white_up - black_up, axis=2)  # type: ignore
```

Comment:
Why?

Reply:
It started failing for me, idk why

Reply:
All of a sudden pyright has a problem with it

Reply:
guess it only is an issue for me locally